### DISCUSSION TITLE
Installing Termux and Andronix Ubuntu with help from CoPilot

### DISCUSSION THREAD

**Discussion by TelsaSecureNet** [termux/termux-app#3924] [General]
So I've been learning to use termux passed 3 to 4 days. I've learned so much on it because I have placed massive amounts of time into it trying to install Ubuntu. Now I saw videos and read and not much of help but I'll tell you this, what I have learned is the AI app Copilot has taught me more than anything. You just have to know how to ask for commands and help. now I was able to get andronix.sh to work for about an hour on vncviewer but then it crashed and now trying to resolve. At first I installed several versions of Linux but it wasn't working correctly and it would crash. I finally reinstalled Android Ubuntu xfce version and it worked for an hour like I said before it crashed. Now can anyone tell me if I should use my Samsung tab s9 plus or use the Samsung s7fe like Ive been doing?

**Comment by twaik** [comment]
This is a termux support space. If your issue is with a third-party project built on top of termux, we probably can't help you with it. Seek help from the author(s) of that project instead.

Third-party projects, with starting points for support where we know them:
- Mobox: <https://github.com/olegos2/mobox/issues>
- Andronix: <https://github.com/AndronixApp/AndronixOrigin/issues>
- TUR (termux-user-repository): <https://github.com/termux-user-repository/tur/issues>

This list isn't complete - if someone tells you a project is a derivative of termux, the same applies. If you need more help contacting the authors of a third-party project, please google it instead of asking for that here.

**Reply by TelsaSecureNet** [reply]
Actually my question was more on the lines of if I'm using termux, would it be more viable to use my Samsung tab s9 plus for performance other than the Samsung tab S7 FE so that termux can work a little better?

**Reply by TelsaSecureNet** [reply]
Sorry I'm just not sure where to ask this question so I thought I would start here.
